### PR TITLE
Add workflows that run later in the pipeline

### LINKED ISSUE #175: :) AMAZING extention, i just have a question
why there are some stuff that are appyed ot top? like face restoreation or Adetailer for example.
I wish there were a way to use Comfy nodes as last step in the chain, the post/pre selector doesnt help in this case here

### PR DESCRIPTION
**Pull request by PladsElsker**
Adds 2 workflows:
- `Postprocess image`
- `Before save image`

There are a couple of things that can run after `postprocess_batch_list`, notably the `face restoration` and the `color corection` sections. 

These workflows cannot edit the size of the batch, but they run later in the pipeline. 

closes #175

### CODE REVIEW

**Comment by PladsElsker** [comment]
Potential improvement to the node amount validation are (checked = done):
- [x] Only check for enabled (not greyed out) nodes
- [ ] Only check for nodes that are connected to a workflow graph that will run

**Comment by John-WL** [comment]
I also don't know why the tests are failing. Is it my code or it's an update from Webui / ComfyUI?

File: comfyui_custom_scripts/extensions/webuiNodes.js [hunk 1]
```diff
@@ -85,8 +85,8 @@ app.registerExtension({
             getTypesLength(iframeInfo.webuiIoTypes.outputs),
             getTypesLength(iframeInfo.webuiIoTypes.inputs),
         );
-        // 240 and 40 are empirical values that seem to work
-        node.size = [240, 40 + distanceBetweenIoSlots * maxIoLength];
+        // 260 and 40 are empirical values that seem to work
+        node.size = [260, 40 + distanceBetweenIoSlots * maxIoLength];
```
Comment (ljleb) [inline comfyui_custom_scripts/extensions/webuiNodes.js:89]: I'm not sure why we're changing this. Is this a typo?

File: lib_comfyui/default_workflow_types.py [hunk 2]
```diff
@@ -20,27 +20,43 @@
...
-postprocess_latent_workflow_type = external_code.WorkflowType(
-    base_id='postprocess_latent',
-    display_name='Postprocess (latent)',
+postprocess_image_workflow_type = external_code.WorkflowType(
+    base_id='postprocess_image',
+    display_name='Postprocess image',
     default_workflow=external_code.AUTO_WORKFLOW,
-    types='LATENT',
+    types='IMAGE',
+    max_amount_of_io_nodes=[None, 1],
```
Comment (ljleb) [inline lib_comfyui/default_workflow_types.py:40]: have you considered splitting this into two `Optional[int]` fields instead of just one?

**Review by ljleb — APPROVED** [review]

**Comment by ljleb** [comment]
Tests fail because they are calling a function that is apparently restricted to comfyui, even though there is no comfyui process registered.

**Pull request merged**